src/v2/recommender.py

```diff
@@ -0,0 +1,490 @@
+"""Gets Alternate and Companion Components recommendation from the recommendation engine.
+
+It also gives a list of packages that are not known to the recommendation engine for further crowd
+sourcing.
+"""
+
+import json
+import datetime
+import requests
+import os
+from collections import defaultdict
+import re
+import logging
+
+from src.utils import (create_package_dict, get_session_retry, select_latest_version,
+                       LICENSE_SCORING_URL_REST, convert_version_to_proper_semantic,
+                       get_response_data, version_info_tuple, persist_data_in_db,
+                       is_quickstart_majority, post_gremlin)
+from src.v2.models import RecommenderRequest
+from src.v2.stack_aggregator import extract_user_stack_package_licenses
+from src.v2.normalized_packages import NormalizedPackages
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__file__)
+
+
+danger_word_list = [r"drop\(\)", r"V\(\)", r"count\(\)"]
+remove = '|'.join(danger_word_list)
+pattern = re.compile(r'(' + remove + ')', re.IGNORECASE)
+pattern_to_save = r'[^\w\*\.Xx\-\>\=\<\~\^\|\/\:]'
+pattern_n2_remove = re.compile(pattern_to_save)
+
+
+class GraphDB:
```

Comment:
Can this be moved into a separate module as this class has only static methods.

Reply:
I didn't do much refactoring on v2 recommender, it looks as close as v1 with only change in the request model. Probably we can do it latter.